front/components/agent_builder/capabilities/shared/ProcessingMethodSection.tsx

```diff
@@ -147,7 +147,7 @@ export function ProcessingMethodSection() {
         }
       }
     }
-  }, [mcpServerView, serversToDisplay, setValue, sources.in]);
```

Comment:
@JulesBelveze unrelate to this but was a lint warning. Was that intentional or is it fine to add it?

Reply:
@spolu mistake of mine, should be fine adding it 👌🏼